xml/http_cookies.xml

```diff
@@ -359,11 +361,12 @@
     <param pos="0" name="service.product" value="Alteon Web Switch"/>
   </fingerprint>
 
-  <fingerprint pattern="^(OBSID)=.*">
+  <fingerprint pattern="^OBSID=.*">
     <description>Observium</description>
-    <param pos="1" name="cookie"/>
+    <param pos="0" name="cookie" value="OBSID"/>
```

Comment:
I am not sure of the point of these, do they provide anything useful to the user?

Reply:
I've never used them. I've only kept them there for consistency sake.  I think Jon added them for cases where the cookie name contains a value that might be useful to the consumer.  An example I've seen would be 
`CF_CLIENT_FOO_CORP=2;Path=/;Expires=Sun, 02-Aug-2020 00:54:53 UTC`
`wordpress_89ca499ef62aa0b115202a9a7cf04c10=+; expires=Fri, 02-Aug-2019 01:55:35 GMT; path=/`
`BIGipServer~web~pool_SOMEPOOLNAME=386140170.36895.0000; path=/; Httponly; Secure`